index.html

```diff
@@ -81,7 +81,7 @@ <h5>I'm a software developer! I can help you build a product ,feature or website
         <!--Dynamic rendering of cards in card container-->
         <section class="cards">
             <div class="card 1"> 
-                <!-- <div class="popup-window hidden" id="Tonic-popup">
+                <div class="popup-window hidden" id="Tonic-popup">
                     <div class="top-block">
                         <div class="frame1">
                             <h2>Tonic</h2>
```

Comment:
- [X] Please make sure to remove the entire work section card from the main page, Work section cards should be created dynamically when the page loads with the information you stored in the javascript array as requested in the project requirements. 👍 👍 

> <img src="https://user-images.githubusercontent.com/83803414/208908439-d96a564b-9849-4772-8e5b-a114c3cf8c82.png">